src/v2/Apps/Partner/Components/PartnerArtists/partnerArtistsUtils.ts

```diff
@@ -93,3 +94,15 @@ export function groupArtists(
     .filter(group => group.artists.length > 0)
     .sort((a, b) => b.artists.length - a.artists.length)
 }
+
+export function getBriefArtistInfo(artist: PartnerArtistDetails_artist) {
```

Comment:
You actually don't need to do this here-- this logic is encapsulated in the `formattedNationalityAndBirthday` field that [already exists on the artist schema](https://github.com/artsy/metaphysics/blob/a1d8173d7b788dd18ec8578524cb5c9a4118ef66/src/schema/v2/artist/index.ts#L532-L551). 

If the logic seems slightly different from the specs, that's OK-- it's better for us to use this shared field and update the logic globally later than to introduce drift.

Reply:
Missed this field, thanks!